pkg/controller/operatingsystemconfig/actuator.go

```diff
@@ -156,9 +157,16 @@ func (a *actuator) handleReconcileOSC(_ *extensionsv1alpha1.OperatingSystemConfi
 		extensionsv1alpha1.Unit{Name: "locksmithd.service", Command: ptr.To(extensionsv1alpha1.CommandStop), Enable: ptr.To(false)},
 	)
 
-	var err error
-	if extensionUnits, extensionFiles, err = a.configureNTPDaemon(extensionUnits, extensionFiles); err != nil {
-		return nil, nil, fmt.Errorf("error configuring NTP Daemon: %v", err)
+	if a.extensionConfig.NTP.Daemon != configv1alpha1.None {
```

Comment:
Doesn't it make sense to do this in the existing switch in `configureNTPDaemon`-function?

Reply:
okay 🤷